app/templates/macros.html

```diff
@@ -83,29 +83,35 @@
             <th scope="col" class="w-20 text-center">Výrobce</th>
             <th scope="col" class="w-20 text-center">
                 Přijato
-                <i class="fas fa-info-circle" data-toggle="tooltip" data-placement="auto"
-                   title="Počet dávek přijatých na očkovací místo.">
+                <i class="fas fa-info-circle" data-toggle="tooltip" data-placement="auto" data-html="true"
+                   title="Počet dávek přijatých na očkovací místo.<br>U krajů předpokládáme, že dávky vydané mimo očkovací místa (např. praktickým lékařům) zůstaly ve stejném kraji.">
```

Comment:
You are mixing očkovací místo and kraj. Očkovací místo - OČM is technical term.
Question: If the metric name is "počet přijatých" should we take care only about arrived vaccines and nothing more?

Reply:
It's because the table is used in different pages - vaccination center detail, kraj detail and total stats